handler/file/retrieve.go

```diff
@@ -172,6 +278,8 @@ func (r *filecoinReader) Seek(offset int64, whence int) (int64, error) {
 		newOffset = r.offset + offset
 	case io.SeekEnd:
 		newOffset = r.size + offset
```

Comment:
ahhh my previous comments about `http.ServeContent` doing a seek-end, seek-start to determine size are handled here, from the db, so I guess we never get to the actual reads or retrieve from filecoion

Reply:
Yes, all good.